src/components/List/components/Item.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { ListItem } from "../type";
+import { timeAgo } from "@/utils/time";
+
+interface ItemProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any;
+  className?: string;
+  onClick?: (item: ListItem) => void;
+}
+
+const Item = ({ data, className, onClick }: ItemProps) => {
+  return (
+    <div
+      className={`flex flex-row items-start gap-[19px] py-[9px] ${className}`}
+      onClick={() => onClick?.(data)}
+    >
+      {data.commenterPhoto && (
+        <div className="w-[29px] h-[29px] rounded-[47.5px] bg-tertiary shrink-0 overflow-hidden">
+          <img
+            src={data.commenterPhoto}
+            alt="Avatar"
+            className="w-full h-full object-cover"
+          />
+        </div>
```

Comment:
this is not nextjs, you can use img will dop

Reply:
Updated